html/browsers/browsing-the-web/navigating-across-documents/failure-check-sequence.https.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Sequence of the checks performed against a navigation response</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+'use strict';
+const collect = (win) => {
+  const report = new Promise((resolve) => {
+    if (!win.ReportingObserver) {
+      return;
+    }
+
+    const observer = new win.ReportingObserver(resolve);
+    observer.observe();
+  }).then((reports) => reports[0].toJSON().type);
```

Comment:
The toJSON() seems unnecessary?

Reply:
That's because it *is* unnecessary :) Removed.